### DISCUSSION TITLE
OpenHantek6022 with Win7-32Bit

### DISCUSSION THREAD

**Discussion by MaMiKoe** [OpenHantek/OpenHantek6022#260] [Q&A] [answered]
I have tried OpenHantek-Win-x86-Release-b488.zip wit nearly a dozend Win7-32Bit-PC (some PC, some Notebook), because none of my home-used PC can run 64Bit-OS nore Win10 (and there seems to be no solution for runing with WinXP until now). Win7 is SP1 and the lastversion/updates. After installing vcredist_x86.exe and before installing the driver for 6022BE/BL (or w/o attaching the 6022BL)  the software shows the initial display (no 6022 detected) but after choosing "demo-mode" the software crashes. The details of the error-report are:

_"Problemsignatur:
  Problemereignisname:	APPCRASH
  Anwendungsname:	OpenHantek.exe
  Anwendungsversion:	0.0.0.0
  Anwendungszeitstempel:	5fe48662
  Fehlermodulname:	Qt5Gui.dll
  Fehlermodulversion:	5.11.3.0
  Fehlermodulzeitstempel:	5bfbc88f
  Ausnahmecode:	c0000005
  Ausnahmeoffset:	000529f2
  Betriebsystemversion:	6.1.7601.2.1.0.256.1
  Gebietsschema-ID:	1031
  Zusatzinformation 1:	0a9e
  Zusatzinformation 2:	0a9e372d3b4ad19135b953a78882e789
  Zusatzinformation 3:	0a9e
  Zusatzinformation 4:	0a9e372d3b4ad19135b953a78882e789"_

When running in the command-windows the report is:

_"QWindowsEGLStaticContext::create: Could not initialize EGL display: error 0x3001
QWindowsEGLStaticContext::create: When using ANGLE, check if d3dcompiler_4x.dll
is available
Attempted to use GDI functions with a non-opengl32.dll library
QWindowsGLContext::QWindowsGLContext: Unable find a suitable pixel format.
Attempted to use GDI functions with a non-opengl32.dll library
QWindowsGLContext::QWindowsGLContext: Unable find a suitable pixel format.
QOpenGLFunctions created with non-current context"_

After installing the driver - doesn´t matter whether the driver in the ZIP-package or with ZADIG 2.7 are used - and attaching the 6022BL for less than a second the initial display appears and then the software immediately also crashes with the same reports/messages as abvove.

Is there any solution to make the software run with Win7-32Bit?

**Comment by Ho-Ro** [comment, marked as answer]
Nobody else answered - maybe because no one uses this setup. The latest binary builds that are available here are 64-only, because the GitHub Win build doesn't support 32bit anymore.
